src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -387,10 +389,14 @@ export class ShelleyWallet implements WalletInterface {
   }
 
   async clear() {
-    await this.transactionCache?.clear()
+    await this.transactionManager.clear()
```

Comment:
Is the `?.` needed here?

Reply:
The "optional chaining" operator was added previously because `transactionCache` was initialized in the method `initialize` not in the `constructor` as it's now